test/unit/nrm/behaviours/save-and-exit.test.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behaviour = require('../../../../apps/nrm/behaviours/save-and-exit');
+const TestDate = 'Mon 05 Jul 2021 13:28:56 +0100';
+const CorrectDate = '02 August 2021';
+const Route = 'save-and-exit';
+const UserEmail = 'test@test.com';
```

Comment:
I prefer using proper use case emails e.g. mary-jane-parker@homeoffice.gov.uk

I find when people write test or foo everywhere it makes it harder to work out or a lot of cognitive load